test/libsolidity/syntaxTests/inlineAssembly/immutables.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+  function f() public pure {
+    assembly {
+      loadimmutable("abc", 0)
+      setimmutable("abc")
```

Comment:
That's the wrong arguments, isn't it? Not that it matters too much, but still... By the way: will ``x := 4`` work, resp. ``a := x`` work for an immutable ``x`` in inline assembly in the future? Shouldn't be too hard to implement actually.

Reply:
i would not want to implement that for now

Reply:
Ok.